### PR TITLE
Register flutter_reactive_ble

### PR DESCRIPTION
**Pull request by miguelcmedeiros**
Register flutter_reactive_ble

### CODE REVIEW

**Comment by miguelcmedeiros** [comment]
@googlebot I signed it!

**Comment by Hixie** [comment]
cc @goderbauer What was the trick you used to make long file names work on Windows?

**Comment by Hixie** [comment]
@miguelcmedeiros Looks like one of the tests is failing on Mac and Linux.

**Comment by goderbauer** [comment]
For the windows filename length issue: add `-c core.longPaths=true` to the git commands as seen here: https://github.com/flutter/tests/blob/master/registry/flutter_packages.test

**Comment by miguelcmedeiros** [comment]
@Hixie, just by creating this PR we could already see that one of our tests failed when using flutter master (we have been using stable and there it passes) 👍.

@goderbauer, thanks for the long-name-trick.

About the CLA, I am part of a corporate agreement which was used in the past for landing change in flutter/flutter repo (other contributor than myself), and although I mention that I sign it, the check is still red. 
Is this a manual check from someone at Google? Should I do something else?

**Comment by miguelcmedeiros** [comment]
After my last update, I'm getting a strange issue during `flutter pub get`. All import URI's in the example app of the package are unknown:
```
|   error • Target of URI doesn't exist: 'package:flutter_reactive_ble_example/src/ble/ble_device_connector.dart' • example/lib/main.dart:3:8 • uri_does_not_exist
|   error • Target of URI doesn't exist: 'package:flutter_reactive_ble_example/src/ble/ble_scanner.dart' • example/lib/main.dart:4:8 • uri_does_not_exist
|   error • Target of URI doesn't exist: 'package:flutter_reactive_ble_example/src/ble/ble_status_monitor.dart' • example/lib/main.dart:5:8 • uri_does_not_exist
...
```

I'm trying to trigger this locally, but I'm failing miserably. Any idea what might be the root-cause?

**Comment by goderbauer** [comment]
@miguelcmedeiros Looks like you may need a separate `flutter packages get` call in your example directory to get rid of those errors.

**Comment by miguelcmedeiros** [comment]
@goderbauer, is that something that is done in `registry/flutter_reactive_ble.test`?

**Comment by goderbauer** [comment]
I don't think we have an example that does it from there. Most people just put a custom script in their repository and invoke it from the `.test` file, as can be seen here: https://github.com/flutter/tests/blob/master/registry/flutter_packages.test

**Comment by miguelcmedeiros** [comment]
By adding `test=flutter packages get` right after update line it worked when testing locally. It would be great if that is sufficient since I won't need to add the scripts.

**Comment by miguelcmedeiros** [comment]
Any idea why `tests-windows` check failed?

**Comment by goderbauer** [comment]
Cirrus didn't produce any logs about the failure. I restarted the check.

**Comment by miguelcmedeiros** [comment]
All tests-checks passed. Do I need to do something else regarding CLA. I can try again the message.

**Comment by miguelcmedeiros** [comment]
@googlebot I signed it!

**Comment by miguelcmedeiros** [comment]
Windows is failing now with timeout... The package as 74 UTs and locally takes a few seconds to run. Any thoughts on this?

**Comment by Hixie** [comment]
We should add some instrumentation/diagnostics to figure out where the time is going. Windows is taking 3x longer than the other platforms here.

**Comment by CaseyHillers** [comment]
@miguelcmedeiros I am working on reducing the windows runtimes here via sharding the tests. I have https://github.com/flutter/flutter/pull/53433 open for allowing this. Just waiting on review until this repo can be updated to take advantage of it.

**Comment by miguelcmedeiros** [comment]
@CaseyHillers, that's great. Thanks!!

**Comment by goderbauer** [comment]
@miguelcmedeiros Can you rebase this to the latest master and see if that fixes the Windows test issue? Thanks.

**Comment by miguelcmedeiros** [comment]
@googlebot I fixed it.

**Comment by miguelcmedeiros** [comment]
@goderbauer, PR is rebased and issue with CLA is solved.

File: registry/flutter_reactive_ble.test [hunk 1]
```diff
@@ -0,0 +1,8 @@
+contact=miguel.catarino.medeiros@signify.com
+contact=remon.helmond@signify.com
+fetch=git -c core.longPaths=true clone https://github.com/PhilipsHue/flutter_reactive_ble.git tests
+fetch=git -c core.longPaths=true -C tests checkout 741b46ae40228f4de5b7bd0d40bc014c4472a1db
+update=.
+test=flutter packages get
```
Comment (Hixie) [inline registry/flutter_reactive_ble.test:6]: this shouldn't be necessary, doesn't `flutter analyze` take care of it?

**Comment by goderbauer** [comment]
I restarted the tests just to make sure that everything is still passing. When everything is green, let's merge this!

File: registry/flutter_reactive_ble.test (hunk 1 above)
Comment (miguelcmedeiros) [reply registry/flutter_reactive_ble.test:6]: I didn't have it initially, but then I got some errors as mentioned in https://github.com/flutter/tests/pull/30#issuecomment-602730131.

After adding this line the errors went away.

**Pull request merged**